src/components/RefineBar/Select.jsx

```diff
@@ -48,7 +48,7 @@ export class Select extends React.Component {
     const values = this.getValues()
 
     return (
-      <section className="cf-select">
+      <section className={'cf-select ' + this.props.id}>
```

Comment:
Should `this.props.id` go in an `id` attribute instead of class?

Reply:
Ah I see in the snapshot it gets repeated. Well, instead of super generic class names like `size` and `sort`, let's prefix this to be `<section className={'cf-select cf-select__' + this.props.id}>`

Reply:
I had to add this here so that I could target these for highlighting in intro.js here

https://github.com/cfpb/ccdb5-ui/pull/409/files#diff-1f017f3f6619b22aabaa7855af7aa91b94892609d8f7f765b5d5c942c4565d98R60

I think it might be possible to update the document.querySelector call to look for attributes instead

Reply:
Hm yea, that actually would be the best way. The prefixed class sets this up as a modifier on `cf-select`, which implies that `cf-select__sort` (or whatever) is modifying `cf-select`. In actuality it is being used as an ID. That's not a big deal, but it could send someone off to look for the modifying CSS. If you want, you should be able to do `<section className="cf-select" data-tour={this.props.id}>` and query it with `document.querySelector('[data-tour="sort"]')`, or `document.querySelector('section[data-tour="sort"]')`.